velox/vector/tests/CMakeLists.txt

```diff
@@ -11,81 +11,87 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(velox_vector_test_lib VectorMaker.cpp VectorTestBase.cpp)
+if(${VELOX_BUILD_TEST_UTILS}
+   OR ${VELOX_BUILD_TESTING}
+   OR ${VELOX_ENABLE_BENCHMARKS_BASIC})
```

Comment:
Its better to have this logic in the parent CMakeLists rather than have this if logic here as it makes it much harder to reason about what gets built etc. 
 
if (${VELOX_ENABLE_BENCHMARKS_BASIC})
  set($VEOX_BUILD_TEST_UTILS ON)
 ..
endif()